bin/merge_md_files.rb

```
@@ -1,45 +1,146 @@
+        files_processed += 1
+      rescue => e
+        puts "  Error processing #{path.basename}: #{e.message}"
       end
     end
-
   end
 
-  puts "Markdown files have been combined into #{output_file}"
+  puts "Generated: #{output_file}"
+  puts "Total sections: #{files_processed}"
 end
```

RuboCop (1.75.5)

[convention] Assignment Branch Condition size for `generate_combined_file` is too high. [<25, 69, 30> 79.28/23]

(Metrics/AbcSize)